kong/clustering/utils.lua

```diff
@@ -5,10 +5,11 @@ local openssl_x509 = require("resty.openssl.x509")
 local ssl = require("ngx.ssl")
 local ocsp = require("ngx.ocsp")
 local http = require("resty.http")
-local ws_client = require("resty.websocket.client")
+local ws_client = require("kong.resty.websocket.client")
 local ws_server = require("resty.websocket.server")
 local utils = require("kong.tools.utils")
 local meta = require("kong.meta")
+local parse_url = require "socket.url".parse
```

Comment:
```suggestion
local parse_url = require("socket.url").parse
```

To follow the style of existing code in this file.